openshift/Dockerfile.recommender

```diff
@@ -0,0 +1,12 @@
+FROM ubi8/nodejs-12
+
+USER 0
+RUN mkdir -p /home/node/app
+COPY base/packages/recommender /home/node/app
+RUN chown -R 1001:0 /home/node/app
+
+USER 1001
+WORKDIR /home/node/app
+ENV HOST=0.0.0.0 PORT=3001
+EXPOSE ${PORT} 50051
+CMD [ "node", "." ]
```

Comment:
Feel free to ignore. Let's add --max-old-space-size perhaps restrict to 1GB or so. 512 which is the default is very small IMO

Reply:
Not done in the Dockerfile.* in the root directory.
I do not want to add something that is not OpenShift related in this PR.

Reply:
If thats the case I recommend you reference the root yaml than duplicate. What do you think?

Reply:
The Dockerfile.* in root did not work for me in OpenShift.

Reply:
Do you mind elaborating on this point?

Reply:
Build terminates with this error if using ./Dockerfile.monorepo instead of openshift/Dockerfile.monorepo in base.yaml:
`error determining run uid: user: unknown user error looking up user "node"`

Reply:
Okay the users were different that means you probably running as root. Its discouraged but I know these Cloud platforms usually limit us

Reply:
OpenShift by default runs containers with random userid for security reasons, not root.
Why do you think it is run as root?

Reply:
Here is a blog post about userid in OpenShift:
https://cookbook.openshift.org/users-and-role-based-access-control/why-do-my-applications-run-as-a-random-user-id.html